docs/openapi.json

```diff
@@ -3198,15 +3198,19 @@
             "schema": {
               "properties": {
                 "input_parameters": {
+                  "description": "Optional. Additional input parameters that override the ones in the workflow specification.",
```

Comment:
Optional. Additional input parameters that override the ones from the workflow specification.